partner-onboarder/README.md

```
@@ -1,48 +1,132 @@
+- S3 bucket name
+- S3 access key
+- S3 secret key
+
+**Option B — NFS**
+Create a directory for the onboarder on the NFS server:
 ```
 mkdir -p /srv/nfs/mosip/<sandbox>/onboarder/
 ```
-* Ensure the directory has 777 permissions:
+Ensure the directory has `777` permissions:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)